### PR TITLE
Minor fixes and adjustments

### PR DESCRIPTION
**Pull request by katzien**
ADD a generic slugify function to always generate slugs in the same way across all the file generators.
FIX a few silly late night bugs from last night - set the bio and github link correctly.
UPDATE Kat's person file

### CODE REVIEW

File: app/src/DataFileGenerator/FileGenerator.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+     * The slugs are file names, and they're also used to link the data together.
+     *
+     * This function replaces any special characters with a hyphen,
+     * replaces spaces with hyphens and lowercases the whole thing.
+     *
+     * @param string $value
+     *
+     * @return string
+     */
+    public function slugify(string $value): string
```
Comment (DaveLiddament) [inline app/src/DataFileGenerator/FileGenerator.php]: I think we should favour composition over inheritance. E.g. have slugify as it's own method (maybe static method on a class) or injected in to the file generators. 
That way slugifier can be reused by things other than FileGenerators and also easy to add tests.

**Review by DaveLiddament — CHANGES_REQUESTED** [review]
See comment about composition over inheritance

File: app/src/DataFileGenerator/FileGenerator.php (hunk 1 above)
Comment (katzien) [reply app/src/DataFileGenerator/FileGenerator.php]: We could do, I suppose slugify would be useful in URL / link construction too. @opdavies are you using anything for that currently?

I think a "utils class" with this function could be a good solution here, then can inject this class via DI.

Comment (opdavies) [reply app/src/DataFileGenerator/FileGenerator.php]: No, I don't think I'm using this anywhere.

Comment (DaveLiddament) [reply app/src/DataFileGenerator/FileGenerator.php]: Yep. I like the utils class injected via DI solution.

**Comment by katzien** [comment]
@DaveLiddament sorry for the delay on this. I moved the slugify function into a StringUtils class and ended up copying Laravel's version of that, since I started adding more and more stuff in and then realised I'm just reinventing the wheel 🤷🏻‍♀️ :)

**Review by DaveLiddament — APPROVED** [review]

**Pull request merged**